dataset.py

```diff
@@ -93,16 +109,28 @@ def collapse_data_dict(data_dict):
     
     return data_dict_collapsed
 
-def read_squad(path, save_dir):
+def read_squad(path, save_dir, orig_source=False, kmeans=False):
```

Comment:
Can we make this 'orig_source' configurable via a command line arg? same with the new kmeans argument